scripts/fetch/fetch.timesteps.R

```diff
@@ -1,14 +1,14 @@
 #' Create a JSON file of timesteps to use in the animation
 fetch.timesteps <- function(viz = as.viz('timesteps')){
   library(dplyr)
-  times <- 
-    # readDepends(viz)[["precip-cell-data"]] %>% # DO THIS WHEN PRECIP IS AVAILBLE
```

Comment:
Since I didn't start from the last viz's code, I guess naming doesn't match. If I'd been thinking I would have output something close to what I was doing before... sorry about that!